extensions/interactions/SetInput/directives/oppia-interactive-set-input.component.spec.ts

```diff
@@ -169,4 +169,22 @@ describe('InteractiveSetInputComponent', () => {
 
     expect(component.answer).toEqual(['test1']);
   });
+
+  it('should set an error message for duplicate answers', () => {
+    const duplicateAnswer = ['item1', 'item1'];
+    component.updateAnswer(duplicateAnswer);
+
+    expect(component.errorMessage)
+      .toBe('Oops, it looks like your answer has duplicates!');
```

Comment:
Same comment as that of Sean's, this needs to be put in the i18n related jsons.

Reply:
I did it. PTAL!